pkg/connection/store/vault/store.go

```diff
@@ -211,7 +212,7 @@ func applyOptions(wo ...store.WriteOption) []kv.ApplyOption {
 				},
 				Data: keyValuesFromData(desired.Data),
 			}
-			if err := o(context.Background(), cs, ds); err != nil {
+			if err := o(ctx, cs, ds); err != nil {
```

Comment:
@turkenh Ditto here - I presume this is safe to pass?

Reply:
👍